src/seq/nucleotide.jl

```diff
@@ -600,30 +693,33 @@ end
 # Mismatch counting
 # -----------------
 
-# compute the mismatch count between two kmers stored in x, y
+@doc doc"Mismatch count between two kmers" ->
 function nucmismatches(x::Uint64, y::Uint64)
     xyxor = x $ y
     return count_ones((xyxor & 0x5555555555555555) | ((xyxor & 0xAAAAAAAAAAAAAAAA) >>> 1))
 end
 
-# return a mask of the first k bits of a Uint64
+@doc doc"Mask of the first `k` bits of a Uint64" ->
 function makemask(k::Integer)
     return 0xffffffffffffffff >> (64 - k)
 end
 
-# Return the number of mismatches between a and b.
-#
-# If a and b are of differing lengths, only the first min(length(a), length(b))
-# nucleotides are compared.
-#
-# Args:
-#   a: first sequence to compare
-#   b: second sequence to compare
-#   nbatches: if true, N matches anything, if false, N matches only itself.
-#
-# Returns:
-#   The number of mismatches.
-#
+@doc doc"""
+`mismatches(a::NucleotideSequence, b::NucleotideSequence, [nmatches=false])`
+
+Return the number of mismatches between `a` and `b`.
+
+If `a` and `b` are of differing lengths, only the first `min(length(a), length(b))`
+nucleotides are compared.
+
+# Arguments
+* `a`: first sequence to compare
+* `b`: second sequence to compare
+* `nbatches`: if true, N matches anything, if false, N matches only itself.
```

Comment:
Random comments are welcome! Specially when they pont out typos and inconsistencies.

n**m**atches isn't required, and the docs should say so.